README.md

```diff
@@ -59,11 +61,20 @@ module.exports = function(config){
 }
 ```
 
-As you can see, the config file is *not* a simple JSON file, it should be a common js (a.k.a. npm) module.  
-You might recognize this way of working from the karma test runner.
+As you can see, the config file is *not* a simple JSON file. It should be a common js (a.k.a. npm) module. You might recognize this way of working from the karma test runner.
 
 Make sure you *at least* specify the `files` and the `testRunner` options when mixing the config file and/or command line options.
 
+## Command-line interface
+
+Stryker can also be installed, configured and run using the [Stryker-CLI](https://github.com/stryker-mutator/stryker-cli). If you plan on using Stryker in more projects, the Stryker-CLI is the easiest way to install, configure and run Stryker for your project.
```

Comment:
Maybe we can add some information here about the working of `stryker-cli`. Mainly that it is only used as a proxy to the local `stryker` instance.